module/commands/report.py

```diff
@@ -12,27 +12,27 @@ def report(update: Update, context: CallbackContext) -> None:
         update: update event
         context: context passed by the handler
     """
-    #getting chat_id and User object of the sender
     chat_id = update.message.chat_id
     chat_user = update.message.from_user
+    user_report_message = update.message.text
 
     if not chat_user.username:
-        context.bot.sendMessage(chat_id=chat_id, text = "")
+        context.bot.sendMessage(chat_id=chat_id, text="Errore! Imposta un username!")
     else:
```

Comment:
there is nothing after the if, we can avoid nesting using return instead of else